tests/background.py

```diff
@@ -292,10 +292,10 @@ def testTicket1781(self):
         bctrl.getStatisticsControl().setNumSigmaClip(10.0)  
         bctrl.getStatisticsControl().setNumIter(1)
         backobj = afwMath.makeBackground(parabimg, bctrl)
-
-        #parabimg.writeFits('in.fits')
-        #backobj.getImageF().writeFits('out.fits')
-
+        
```

Comment:
To clarify, since it might not be obvious in the diff -- you do want a blank line here, but you currently have a line containing 8 space characters. It's them that I suggest you remove, not the line itself.